common/build.gradle

```diff
@@ -2,6 +2,8 @@ apply plugin: 'com.android.library'
 apply plugin: 'pmd'
 apply plugin: 'checkstyle'
 apply plugin: 'maven-publish'
+//Can enable when we determine that desugaring works for this lib.
+//apply plugin: 'de.mannodermaus.android-junit5'
```

Comment:
This looks like something we would need an OSS Component Review for if we enable it. Do we want to include this in this PR (even in a commented-out state?)